app/lib/pdf_filler/ny215_pdf.rb

```diff
@@ -41,19 +41,19 @@ def hash_for_pdf
         'Worksheet B 4 dollars15' => claimed_attr_value('E_TOT_OTHCR_AMT'),
         'Worksheet B 5 dollars15' => claimed_attr_value('E_NET_TX_AMT')
       }
-      @xml_document.css('dependent').each_with_index do |dependents_node, index|
+      @submission.data_source.dependents.select{ |dep| dep.eligible_for_eitc }.each_with_index do |dependents_node, index|
         index += 1
         answers.merge!({
-                         "ln34fn#{index}" => dependents_node.at("DEP_CHLD_FRST_NAME")&.text,
-                         "ln3mi#{index}" => dependents_node.at("DEP_CHLD_MI_NAME")&.text,
-                         "ln34ln#{index}" => dependents_node.at("DEP_CHLD_LAST_NAME")&.text,
-                         "ln34suf#{index}" => dependents_node.at("DEP_CHLD_SFX_NAME")&.text,
-                         "ln34real#{index}" => dependents_node.at("DEP_RELATION_DESC")&.text,
-                         "month#{index}" => dependents_node.at("DEP_MNTH_LVD_NMBR")&.text,
-                         "ln34disability#{index}" =>  dependents_node.at("DEP_DISAB_IND")&.text == '1' ? "Yes" : "Off",
-                         "ln34student#{index}" =>  dependents_node.at("DEP_STUDENT_IND")&.text == '1' ? "Yes" : "Off",
-                         "ln34ssn#{index}" => dependents_node.at("DEP_SSN_NMBR")&.text,
-                         "ln34birth#{index}" => (Date.parse(dependents_node.at("DOB_DT")&.text)).strftime("%m%d%Y")
+                         "ln34fn#{index}" => dependents_node.first_name,
+                         "ln3mi#{index}" => dependents_node.middle_initial,
+                         "ln34ln#{index}" => dependents_node.last_name,
+                         "ln34suf#{index}" => dependents_node.suffix,
+                         "ln34real#{index}" => dependents_node.relationship,
+                         "month#{index}" => nil,
+                         "ln34disability#{index}" =>  dependents_node.eic_disability,
+                         "ln34student#{index}" =>  dependents_node.eic_student ? "Yes" : "Off",
+                         "ln34ssn#{index}" => dependents_node.ssn,
+                         "ln34birth#{index}" => dependents_node.dob.strftime("%m%d%Y")
```

Comment:
Adds EITC dependents to NY215

Reply:
![Screen Shot 2024-01-02 at 3 22 05 PM](https://github.com/codeforamerica/vita-min/assets/2385700/0dd35f85-2ce7-44b7-bd81-5dc9950cb61b)